### PR TITLE
Fix/TR-3424/graphic PCIs in test review

### PR DESCRIPTION
**Pull request by bziondik**
Related to: https://oat-sa.atlassian.net/browse/TR-3424

Depends on:

- [x] https://github.com/oat-sa/qti-sdk/pull/339 - `fix/TR-3424/unmarshall-list-inside-record-container`
- [ ] taoOutcomeUi - `feature/TR-3424/handle-record-type-response-variable`
- [ ] lib-tao-qti - `feature/TR-3424/transform-record-to-database-store`

Related issue: https://github.com/oat-sa/extension-tao-itemqti/pull/2255 - popups in PCI authoring

Fix review mode for these PCI: update `destroy` and `getInstance` function.
**Root of issue:** in review we render PCI 2 times, one initial render, then destroy and render with state again. `Destroy` removed all markup - and second render fails because no elements in DOM. Also was no call to `setResponse` in `getInstance` function.

Also was updated responses record for some of the PCI, to have proper structure.

**How to test:**

1. install extension
2. switch `taoOutcomeUi` and 2 libs `vendor/tao-sa/lib-tao-qti` and `vendor/qtism/qtism` to proper branches
3. create items with all PCIs (6 items)
4. create test
5. create delivery
6. pass delivery
7. go to Results page
8. view delivery execution, check that all Responses set in items Review (click Review button)

image

image

### CODE REVIEW

File: views/js/pciCreator/ims/graphFunctionInteraction/runtime/graphFunctionInteraction.js [hunk 1]
```diff
@@ -181,9 +181,12 @@ define([
          * @param {Object} [state] - the json serialized state object, returned by previous call to getStatus(), use to initialize an
          */
         getInstance : function getInstance(dom, config, state){
-            var response = config.boundTo;
+            var boundTo = config.boundTo;
```
Comment (andreluizmachado) [inline views/js/pciCreator/ims/graphFunctionInteraction/runtime/graphFunctionInteraction.js:184]: can't we use const here?

Comment (andreluizmachado) [reply views/js/pciCreator/ims/graphFunctionInteraction/runtime/graphFunctionInteraction.js:184]: Th ci seems to be broken as well. The is unknown for me. Probably @wazelin can help on this.

**Review by Silvia-loza — APPROVED** [review]
Great job! :clap:
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [X] New code is respecting code style rules
- [X] New code is respecting best practices
- [X]  Pull request's target is not ` master`
- [ ] Feature is working correctly on my local machine (if applicable)
- [X] Acceptance criteria are respected
- [X] Pull request title and description are meaningful

**Review by marpesia — APPROVED** [review]
**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Commits messages are meaningful
- [x] Commits are atomic
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

File: views/js/pciCreator/ims/graphFunctionInteraction/runtime/graphFunctionInteraction.js (hunk 1 above)
Comment (wazelin) [reply views/js/pciCreator/ims/graphFunctionInteraction/runtime/graphFunctionInteraction.js:184]: Fixed the sniffer introduced by https://github.com/oat-sa/tao-extension-ci-action/releases/tag/v1.1.0 in the latest release.

**Pull request merged**